packages/linter/src/builders/lint/lint.impl.ts

```diff
@@ -1,23 +1,368 @@
-import {
-  BuilderContext,
-  BuilderOutput,
-  createBuilder
-} from '@angular-devkit/architect';
+import { createBuilder } from '@angular-devkit/architect';
+import eslint, { CLIEngine } from 'eslint';
+import { existsSync, readFileSync, writeFileSync } from 'fs';
+import * as glob from 'glob';
+import { Minimatch } from 'minimatch';
+import * as path from 'path';
+import * as ts from 'typescript';
+import { BuilderContext, BuilderOutput } from '@angular-devkit/architect';
 import { from, Observable } from 'rxjs';
-import { concatMap } from 'rxjs/operators';
+
```

Comment:
Possibly worth adding a comment that we've adapted this from the angular-eslint repo?